File: lab5/lab5.py
from os import system
import math

print("The list of three-digit numbers, which are equal to the sum of their digits, powered to 3:")

for hundred in range(1, 10):
    for ten in range(0, 10):
        for one in range(0, 10):
            sum = pow(hundred, 3) + pow(ten, 3) + pow(one, 3)
            number = hundred * 100 + ten * 10 + one
            
            if sum == number:
                print(number)

File: lab5/lab5.cpp
#include <iostream>
#include <math.h>
using namespace std;

int main()
{
cout << "The list of three-digit numbers, which are equal to the sum of their digits, powered to 3:" << endl;

for(int hundred = 1; hundred <= 9; hundred++)
    {
        for(int ten = 0; ten <= 9; ten++)
        {
            for(int one = 0; one <= 9; one++)
            {
                int sum = pow(hundred, 3) + pow(ten, 3) + pow(one, 3);
                int number = hundred * 100 + ten * 10 + one;
                
                if(sum == number)
                {
                    cout << number << endl;
                }
            }
        }
    }
}